lib/fluent/plugin/out_influxdb.rb

```diff
@@ -20,9 +21,10 @@ def initialize
 
   def configure(conf)
     super
-    @influxdb = InfluxDB::Client.new @dbname, host: @host, 
-                                              port: @port, 
-                                              username: @user, 
+    @remove_tag_prefix = conf['remove_tag_prefix']
```

Comment:
I'm not sure I follow? Are you referring to the gsub below expecting a '.'? I simply took that line from the mixpanel plugin. Otherwise, I don't understand what you mean.